Resources/Prototypes/_NF/Entities/Objects/Fun/spaceblade.yml

```diff
@@ -0,0 +1,249 @@
+- type: entity
+  id: SpaceBlade
+  parent: AirTank
+  name: space blade
+  description: Let it rip
+  abstract: true
+  components:
+  - type: GasTank
+    outputPressure: 0.1
+    maxOutputPressure: 0.1
+    valveOutputRate: 50
+    air:
+      volume: 1
+      moles:
+         - 0.4101137219 # oxygen
+      temperature: 293.15
+  - type: Physics
+    bodyType: Dynamic
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.25,-0.25,0.25,0.25"
+        density: 0.1
+        mask:
+        - SmallMobMask #this is so they can go under plastic flaps
+        layer:
+        - MachineLayer
```

Comment:
Is there a layer where the player doesn't step on these but they'll interact with each other?

Reply:
I wise

Reply:
Would BulletImpassable be enough on its own?

Reply:
Hear me out here.
Mask: InteractImpassible
Layer: InteractImpassible.

Cons: it can roll right overtop of machines and looks kind of goofy
Pros: it interacts with itself, walls, glass, doors, etc, doesn't block or interact with people at all and doesn't block bullets.

Reply:
Its ok, fixed